src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -475,6 +491,11 @@ contract SecurityCouncilNomineeElectionGovernor is
         );
     }
 
+    /// @notice Deprecated, use `addContender(uint256 proposalId, bytes calldata signature)` instead
+    function addContender(uint256) external pure {
```

Comment:
we can have 2 optional entrypoints.
addContederImpl that is called with prop id and sender
the signature entrypoint and msg.sender entrypoint both call it.

keeps the code backward compatible

Reply:
Depends on the motivation behind the change; if we want to guarantee that the sender can produce a signature, then we may want to disallow the old addContender (allows for smart contract wallet, address alias, etc. that can't necessarily sign on all governed chains). 

if the motivation is just to add a new feature, then we keep it

Reply:
thats true, lets not keep it to ensure that by not keeping it backwards compatible.
would be good to add comments alongside the code that explain the above

Reply:
> thats true, lets not keep it to ensure that by not keeping it backwards compatible.

so just to be sure – keep the logic as it currently is on this branch (`addContender(uint256)` method still exists but just reverts with a deprecation error) but add more natspec comments explaining the rationale (restrict contender-adding to only an address that can produce a sig)